sysbox-in-docker/Dockerfile.debian-bullseye

```diff
@@ -0,0 +1,59 @@
+#
+# Sysbox-In-Docker Container Dockerfile (Debian-Bullseye image)
+#
+# This Dockerfile creates the sysbox-in-docker container image, which holds
+# all Sysbox binaries and its dependencies. The goal is to allow users to run
+# an entire Sysbox sandbox within a container.
+#
+# Notice that the image must be run as a privileged container, and a few resources
```

Comment:
I would reword this as:

NOTE: Sysbox is a container runtime and thus needs host root privileges. As a result, this image must be run as a privileged container ...  in inner containers. Note that within the privileged container, inner containers launched with Docker + Sysbox will be strongly isolated from the host by Sysbox (e.g., via the Linux user-namespace).

Reply:
Done.